line.js

```diff
@@ -1,47 +1,47 @@
-/*
+/**
  * Line Utils
  * A line is an array of two points e.g. [[-5, 0], [5, 0]].
+ *
+ * @flow
```

Comment:
I kind of like having the part of the doctoring instead it being a separate comment.  Nice!

Reply:
I've seen it in other libraries and it feels a lot cleaner than:
```js
// @flow 
/* Line utils
 * A line is an array...
 */
```